libretroshare/src/rsserver/p3face.h

```diff
@@ -188,16 +188,13 @@ class RsServer: public RsControl, public RsTickingThread
 
 		// Worker Data.....
 
-    int mMin ;
-    int mLoop ;
-    int mLastts ;
-    long mLastSec ;
-    double mAvgTickRate ;
-    double mTimeDelta ;
+        double mLastts;
+        double mTickInterval;
```

Comment:
should use <TAB> for indentation instead of space, like the following two lines did you read this  https://retroshare.readthedocs.io/en/latest/developer/coding/  ?